src/platform.ts

```diff
@@ -47,8 +64,6 @@ export class MiHomeGateway implements DynamicPlatformPlugin {
   configureAccessory(accessory: PlatformAccessory) {
     this.log.debug('Restoring accessory from cache:', accessory.displayName);
 
-    this.createAccessoryHandler(accessory);
```

Comment:
I'm guessing this is superfluous - could you just outline why this is no longer required?

Reply:
Creating the accessory handler for new and existing accessories is handled in `discoverDevices`.
(Not sure how I reference the lines here).

Reply:
Thanks for spotting this!